diff --exclude=.git --exclude=.hg -rupN example/demo.c example_new/demo.c
--- example/demo.c	2024-11-28 20:44:24
+++ example_new/demo.c	2024-11-25 11:41:56
@@ -5,9 +5,9 @@
 #ifdef NANOVG_GLEW
 #  include <GL/glew.h>
 #endif
-#include <GLFW/glfw3.h>
+//#include <GLFW/glfw3.h>
 #include "nanovg.h"
-#define STB_IMAGE_WRITE_IMPLEMENTATION
+//#define STB_IMAGE_WRITE_IMPLEMENTATION
 #include "stb_image_write.h"
 
 
@@ -1214,6 +1214,7 @@ void saveScreenShot(int w, int h, int premult, const c
 
 void saveScreenShot(int w, int h, int premult, const char* name)
 {
+#if 0
 	unsigned char* image = (unsigned char*)malloc(w*h*4);
 	if (image == NULL)
 		return;
@@ -1225,4 +1226,5 @@ void saveScreenShot(int w, int h, int premult, const c
 	flipHorizontal(image, w, h, w*4);
  	stbi_write_png(name, w, h, 4, image, w*4);
  	free(image);
+#endif
 }
diff --exclude=.git --exclude=.hg -rupN example/perf.c example_new/perf.c
--- example/perf.c	2024-11-28 20:45:25
+++ example_new/perf.c	2024-11-25 11:41:56
@@ -1,3 +1,4 @@
+// clang-format off
 #include "perf.h"
 #include <stdio.h>
 #include <string.h>
@@ -5,7 +6,7 @@
 #ifdef NANOVG_GLEW
 #  include <GL/glew.h>
 #endif
-#include <GLFW/glfw3.h>
+//#include <GLFW/glfw3.h>
 #include "nanovg.h"
 
 #ifdef _MSC_VER
@@ -41,12 +42,15 @@ void startGPUTimer(GPUtimer* timer)
 
 void startGPUTimer(GPUtimer* timer)
 {
+#if 0
 	if (!timer->supported)
 		return;
 	glBeginQuery(GL_TIME_ELAPSED, timer->queries[timer->cur % GPU_QUERY_COUNT] );
 	timer->cur++;
+#endif
 }
 
+#if 0
 int stopGPUTimer(GPUtimer* timer, float* times, int maxTimes)
 {
 	NVG_NOTUSED(times);
@@ -72,6 +76,7 @@ int stopGPUTimer(GPUtimer* timer, float* times, int ma
 	}
 	return n;
 }
+#endif
 
 
 void initGraph(PerfGraph* fps, int style, const char* name)
@@ -184,3 +189,5 @@ void renderGraph(NVGcontext* vg, float x, float y, Per
 		nvgText(vg, x+w-3,y+3, str, NULL);
 	}
 }
+
+// clang-format on
